public/index.html

```diff
@@ -5,10 +5,6 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
```

Comment:
Hvorfor ikke ha noe fornuftig? :)

Reply:
Litt usikker på hva vi skl ha :)

Reply:
Ettersom dette gjelder for både søknad om stønad og skjema for arbeidssøker er jeg usikker på hva en beskrivelse burde inneholdt.